docs/components/zeebe/zeebe-overview.md

```diff
@@ -5,21 +5,25 @@ sidebar_label: "Introduction"
 description: "Zeebe is the workflow engine powering Camunda 8. While written in Java, you do not need to be a Java developer to use Zeebe."
 ---
 
-Zeebe is the workflow engine powering Camunda 8. While written in Java, you do not need to be a Java developer to use Zeebe.
+import OverviewImg from '../zeebe/technical-concepts/assets/zeebe-architecture.png';
```

Comment:
🔧 I feel we can update this image as well. Not sure whether we want to do it now

Reply:
Would your team be able to put together a new architecture diagram? If so, I'm happy to drop this in this PR, or I can create a separate issue.

Reply:
We might should do this as part of an effort to show the complete architecture (of the whole platform) wdyt? Meaning including webapps, identity, rest API, etc.